Tests iOS/SignOutTests.swift

```diff
@@ -44,4 +44,35 @@ class SignOutTests: XCTestCase {
         try server.stop()
         app.terminate()
     }
+    
+    func test_tappingSignOutshowsLogin() {
+        app.tabBar.settingsButton.wait().tap()
+        tap_SignOut()
+        
+        let account = XCUIApplication()
+        account.alerts["Are you sure?"].scrollViews.otherElements.buttons["Sign Out"].wait().tap()
+        app.accountView.loginText.exists
+    }
+    
+    func test_tappingSignOutCancelshowsAccountsMenu() {
+        app.tabBar.settingsButton.wait().tap()
+        tap_SignOut()
+        
+        let account = XCUIApplication()
+        account.alerts["Are you sure?"].scrollViews.otherElements.buttons["Cancel"].wait().tap()
```

Comment:
I like what you did for the `settingsButton` and `loginText`, should we do the same here?

Reply:
I actually had trouble getting this to work any other way, but would be open to any suggestions for improvement...

When I tried to add element.alerts["Are you sure?"].scrollViews.otherElements.buttons["Cancel"].wait().tap()
into AccountViewElement, I got an error

Reply:
I also realize our `AccountViewElement` is pointing to the wrong place so that could be the reason why, I can make the change on this branch if you would like or we can do this async. Let me know how you prefer to proceed :) Our old implementation was using `AccountViewController` but now we are using `SettingsViewController`

Reply:
Oh interesting, yes please make any changes you would like to the branch yes. 🙌